deployment/clouddeploy/gke-workers/base/gen-cperepos-map.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: gen-cperepos-map
+spec:
+  schedule: "0 6 */1 * *"
```

Comment:
Does this use what debian_copyright_mirror outputs? Would it be better to schedule this for a bit after it?

Reply:
It does, but I'm not too worried about this being tightly coupled. 

I'll potentially dial down the frequency of the Debian copyright mirroring once it's proved itself, because I don't expect it to significantly change (although for the same reason, continuing to run it daily won't be a big deal).